packages/jszip-cli/src/BuildService.ts

```diff
@@ -73,6 +75,15 @@ export class BuildService {
     return this;
   }
 
+  /**
+   * Note Glob patterns should always use / as a path separator, even on Windows systems,
```

Comment:
```suggestion
   * Note: glob patterns should always use / as a path separator, even on Windows systems,
```

Reply:
That's cool! I had commited the changes. And I think glob@10 works fine without setting `esModuleInterop`.